src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/InitializeHandler.cs

```diff
@@ -22,6 +22,11 @@ internal class InitializeHandler : IRequestHandler<InitializeParams, InitializeR
                     ResolveProvider = true,
                     TriggerCharacters = new[] { ".", "@", "<", "&", "\\", "/", "'", "\"", "=", ":" }
                 },
+                DocumentOnTypeFormattingProvider = new DocumentOnTypeFormattingOptions()
+                {
+                    FirstTriggerCharacter = ">",
+                    MoreTriggerCharacter = new[] { "=" }
```

Comment:
Out of curiosity did you ever figure out the reasoning of having a "More"? Only reason I can really imagine is that they messed the spec up v1 and then added a "More" to account for "First" not being an array lol

Reply:
Not really. I tried looking up online but couldn't find anything. My guess would be they decided to optimize for the most common case (having a single character) so that the clients don't have to search through an array of length 1 all the time?